app/models/whitehall_migration/document_import.rb

```diff
@@ -4,6 +4,8 @@
 # the import status of the document into Content Publisher
 class WhitehallMigration::DocumentImport < ApplicationRecord
   belongs_to :document, optional: true
+  has_many :asset_imports
+  alias_attribute :assets, :asset_imports
```

Comment:
Might be better to name the association assets e.g. `has_many :assets, class: "WhitehallMigration::AssetImport", foreign_key: :asset_import_id`

Reply:
Thanks, was searching for the right syntax to use!

Reply:
Hmmm this is causing some failures - I had to change `class: "WhitehallMigration::AssetImport"` to `class_name: "WhitehallMigration::AssetImport"`, but was getting some transaction PostgreSQL errors like this:

```
     ActiveRecord::StatementInvalid:
       PG::InFailedSqlTransaction: ERROR:  current transaction is aborted, commands ignored until end of transaction block
     # ./lib/whitehall_importer.rb:45:in `rescue in sync'
     # ./lib/whitehall_importer.rb:36:in `sync'
     # ./lib/whitehall_importer.rb:14:in `import_and_sync'
     # ./spec/lib/whitehall_importer_spec.rb:21:in `block (3 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # PG::UndefinedColumn:
     #   ERROR:  column whitehall_migration_asset_imports.asset_import_id does not exist
     #   LINE 1: ....* FROM "whitehall_migration_asset_imports" WHERE "whitehall...
     #                                                                ^
     #   ./lib/whitehall_importer/migrate_assets.rb:16:in `reject'
```

Also tried renaming `foreign_key: :whitehall_migration_asset_import_id` but that column doesn't exist either.

Reply:
Sure, maybe come grab me about that if you're still about the office and we can work it out.

Reply:
For the record, `foreign_key: :asset_import_id` wasn't needed. Omitting it made it work 🎉